### PR TITLE
version bump and partisan upgrade

### PR DESCRIPTION
**Pull request by tsloughter**

### CODE REVIEW

**Comment by tsloughter** [comment]
Hey @lrascao could you take a look at this failure? Chris suggested asking you.

**Comment by lrascao** [comment]
yeah, this is due to the probabilistic nature of the hyparview membership, in some runs you might not get a connected graph, i've talked about this with Chris in the past, i think in Lasp they check for this and request a rebuild of the overlay.

**Comment by cmeiklejohn** [comment]
So, this test is nondeterministic because the overlay isn't guaranteed to be connected?  I'd like to see it pass first before we confirm that is the actual cause/problem/solution.

**Comment by lrascao** [comment]
i think that's the case yes, it's running again now

**Comment by tsloughter** [comment]
Cool. Yea, I just pushed a change so it would run on otp 20, which kicked it off again.

File: rebar.config [hunk 1]
```diff
@@ -17,6 +17,9 @@
 {edoc_opts, [{preprocess, true}]}.
 
 {profiles, [
+    {test, [
```
Comment (cmeiklejohn) [inline rebar.config:20]: It would be great to have this on Partisan and Lasp so we could run Erlang 20 builds there too...

Comment (tsloughter) [reply rebar.config:20]: k

**Comment by cmeiklejohn** [comment]
@lrascao send me your email so we can invite you to the slack channel

**Comment by lrascao** [comment]
it's on my profile: luis.rascao@gmail.com, thanks!

**Pull request merged**